models/engine/db_storage.py

```diff
@@ -74,3 +74,16 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """Returns the object based on the class name and its ID"""
+        classData = self.__session.query(cls).filter(cls.id == id).first()
```

Comment:
Why do you use a filter for looking for a class id? that can result in a waste of resources. For that you need the first(), but even though is a waste. What if you use the one_or_none () it is more clear and useful for this function